src/DropKit.ts

```diff
@@ -66,69 +69,88 @@ export default class DropKit {
 
       this.walletAddress = await signerOrProvider.getAddress()
       this.contract = new ethers.Contract(data.address, abi, signerOrProvider)
+      if (!this.contract) {
```

Comment:
Noticed that this validation was done in several places, decided to keep it on the initialization of the DropKit rather than repeating it on each method